CONTRIBUTE.md

```diff
@@ -0,0 +1,333 @@
+# JavaScript Interactive Examples Contributing Guide
+
+A guide for people wanting to contribute interactive JavaScript examples.
+
+## Naming Things
+
+It probably makes sense to name things in a consistent way throughout all examples.  Where possible the naming should ‘reflect’ the type of that object or variable.
+
+### Numerical Suffixes
+
+One way we could maintain a certain consistency is to use the type of the object with a numerical suffix.
+
+For example:
+
+```
+object1 = {};
+var string1 = “a string”;
+var array1 = ['a', 'b', 'c'];
+var array2 = [1, 2, 3];
+```
+
+Note it is suggested that even if there is only one instance of that object then it should be suffixed with a `1`.
+
```

Comment:
I find language like this a bit too wishy washy. I think you should be more prescriptive and direct, and ummm, less English ;-)

So for example

"Note that even if there is only one instance of that object, it should be suffixed with a `1`."

Reply:
Right-ho! ;)